frontend/src/pages/Contracts.tsx

```diff
@@ -8,7 +8,15 @@ const Contracts = () => {
   return (
     <>
       <SortFilterHeading heading="Contracts" displayCreate />
-      {isLoading ? <div>Loading..</div> : <ContractsContainer jobs={jobs} />}
+      {isLoading ? (
+        <div>Loading..</div>
+      ) : jobs.length ? (
+        <ContractsContainer jobs={jobs} />
+      ) : (
+        <div>
+          <div>No jobs are available to show at this time.</div>
```

Comment:
Probably don't need this extra wrapping div, right?

Also, maybe this message is better as `No jobs found.`